src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class `RightTriangle` should not begin with an empty line according to the checklist. Please remove the empty line at the beginning of the class definition.